### PR TITLE
Improve output documentation

### PR DESCRIPTION
**Pull request by d4straub**
Output documentation needed a review. Here it is.

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/mag branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/mag)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [x] `README.md` is updated

### CODE REVIEW

**Comment by d4straub** [comment]
Since local linting and linting in my repo passes, I'll trigger it here once more.

File: docs/output.md [hunk 1]
```diff
@@ -188,13 +197,35 @@ All the files in this folder contain small and/or unbinned contigs that are not
...
+**Output directory: `results/GenomeBinning/QC/QUAST/[assembler]-[bin]`**
+
+- `report.*`: QUAST report in various formats, such as html, txt, tsv or tex
+- `quast.log`: QUAST log file
+- `predicted_genes/[assembler]-[sample].rna.gff`: Contig positions for rRNA genes in gff version 3 format
+
+**Output directory: `results/GenomeBinning/QC`**
+
+- `quast_summary.tsv`: QUAST output for all bins summarized
+- `quast_and_busco_summary.tsv`; Summary of BUSCO and QUAST results
```
Comment (skrakau) [inline docs/output.md:213]:
```suggestion
- `quast_and_busco_summary.tsv`: Summary of BUSCO and QUAST results
```

File: docs/output.md [hunk 2]
```diff
@@ -27,40 +27,41 @@ For further reading and documentation see the [FastQC help](http://www.bioinform
...
+- `[sample]_remove_phix_log.txt`: Contains a brief log file indicating how many reads have been retained.
 
 ### Host read removal
 
 The pipeline uses bowtie2 to map short reads against the host reference genome specified with `--host_genome` or `--host_fasta` and removes mapped reads.
 
 **Output directory: `results/QC_shortreads/remove_host`**
 
-- Contains the bowtie2 log file indicating how many reads have been mapped as well as a file listing the read ids of discarded reads.
+- `[sample].bowtie2.log`: Contains the bowtie2 log file indicating how many reads have been mapped as well as a file listing the read ids of discarded reads.
```
Comment (skrakau) [inline docs/output.md:56]:
```suggestion
- `[sample].bowtie2.log`: Contains the bowtie2 log file indicating how many reads have been mapped as well as a file listing the read ids of discarded reads.
```

The information about discarded and kept reads is also included in the MultiQC report

File: docs/output.md [hunk 3]
```diff
@@ -71,111 +72,119 @@ However, since within this pipeline filtlong uses a read quality based on k-mer
...
+- `[sample]_graph.gfa.gz`: Compressed assembly graph in gfa format
+- `[sample]_contigs.fasta.gz`: Compressed assembled contigs in fasta format
+- `[sample]_log.txt`: Log file
+- `[sample]_QC/`: Directory containing QUAST files
 
-### Quast
+### Metagenome QC with QUAST
 
-[quast](http://cab.spbu.ru/software/quast/) is a tool that evaluates genome and metagenome assemblies by computing various metrics
+[QUAST](http://cab.spbu.ru/software/quast/) is a tool that evaluates metagenome assemblies by computing various metrics. The QUAST output is also included in the multiqc report, as well as in the assembly directories themselves.
```
Comment (skrakau) [inline docs/output.md:160]:
```suggestion
[QUAST](http://cab.spbu.ru/software/quast/) is a tool that evaluates metagenome assemblies by computing various metrics. The QUAST output is also included in the MultiQC report, as well as in the assembly directories themselves.
```

File: docs/output.md [hunk 4]
```diff
@@ -27,40 +27,41 @@ For further reading and documentation see the [FastQC help](http://www.bioinform
...
 
-[fastp](https://github.com/OpenGene/fastp) is a all-in-one fastq preprocessor for read/adapter trimming and quality control. It is used in this pipeline for trimming adapter sequences and discard low-quality reads.
+[fastp](https://github.com/OpenGene/fastp) is a all-in-one fastq preprocessor for read/adapter trimming and quality control. It is used in this pipeline for trimming adapter sequences and discard low-quality reads. Output is in the results folder and part of the MultiQC report.
 
-**Output directory: `results/QC_shortreads/fastp`**
+**Output directory: `results/QC_shortreads/fastp/[sample]`**
 
-- The trimmed reads are not included in the output by default, but the `--keep_trimmed` will store the trimmed reads in this directory
+- `fastp.html`: Interactive report
+- `fastp.json`: Report in json format
```
[the remark this answers is not in the document]
Comment (d4straub) [reply docs/output.md:40]: I actually checked, that parameter doesn't exist.

File: docs/output.md (hunk 2 above)
Comment (d4straub) [reply docs/output.md:56]: Added this.

**Pull request merged**